sktime/registry/_tags.py

```diff
@@ -104,12 +104,6 @@
         "type",
         "passed to input checks, input conversion index type to enforce",
     ),
-    (
```

Comment:
have you checked whether this is not used in forecasters anymore?

Reply:
yes